src/Styleguide/StyledSystemTypes.ts

```diff
@@ -0,0 +1,26 @@
+export interface MarginProps {
+  m?: number
+  mr?: number
+  ml?: number
+  mt?: number
+  mb?: number
+}
+
+export interface FlexboxProps {
+  flexDirection?: "row" | "column"
+  alignItems?: "flex-start" | "flex-end" | "center" | "baseline" | "stretch"
+  alignContent?:
+    | "flex-start"
+    | "flex-end"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "stretch"
+  justifyContent?:
+    | "flex-start"
+    | "flex-end"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "space-evenly"
+}
```

Comment:
@zephraph - wanted to point this out as @yuki24 has already typed much of this in his spike work: https://github.com/artsy/emission/blob/master/src/lib/Components/Bidding/Elements/types.d.ts

Reply:
I'm sure some incorrect definitions slipped in due to a human error... I wonder if there's any better way to auto-generate all these type definitions.

cc @alloy

Reply:
Oh nice! Good catch. Can we share that? 😄

Reply:
@yuki24 Are these typings for the styled-system API? If so, no typings are maintained manually, but you should consider publishing them to http://github.com/DefinitelyTyped/DefinitelyTyped instead so others can help maintaining them.

Reply:
Right. Publish it on DT so we are at least maintainers and then invite that person to be a maintainer too?

Reply:
Once I get a little farther on the comps stuff I'll pull down and figure out how to add a lib to it; i haven't done that before.